src/database/sqlite/event/event_database.py

```diff
@@ -1450,14 +1440,15 @@ def _write_stored_tournament(
         self,
         stored_tournament: StoredTournament,
     ) -> StoredTournament:
+        per_plugin_player_data = plugin_manager.hook.tournament_data_for_db_write(stored_tournament=stored_tournament)
+        plugin_data = { key: value for data in per_plugin_player_data for key, value in data.items() }
```

Comment:
```suggestion
        plugin_data = {key: value for data in per_plugin_player_data for key, value in data.items()}
```
no space after a `{` or before a `}` is a coding standard:  
![image](https://github.com/user-attachments/assets/a1f8cf89-0560-4e75-9c8f-b09d784fa97a)